Resources/Prototypes/_EE/Entities/NanoTrasen/Modsuits/inquisitory.yml

```
@@ -0,0 +1,163 @@
+    - type: FlashImmunity # Goobstation
+    - type: FlashSoundSuppression # Goobstation
+      protectionRange: 0
+    - type: ShowHealthBars
+      damageContainers:
+        - Biological
+    - type: PointLight
+      color: "#FFDF00"
+    - type: SealableClothingVisuals
+      visualLayers:
+        head:
+        - state: equipped-HEAD-sealed
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)